stdlib/3/tkinter/dialog.pyi

```diff
@@ -0,0 +1,10 @@
+from tkinter import *
```

Comment:
Can you also remove this here and in the next file? (Should have been more explicit in my previous message.)

Reply:
If I simply remove these lines, then my code that uses tkinter.filedialog won't typecheck anymore and, more importantly, typeshed's own tests will fail. :) However, to avoid confusion, I've replaced `from tkinter import *` with explicit imports.